composer.json

```diff
@@ -17,10 +17,10 @@
         }
     },
     "require": {
-        "php" : ">=7.2"
+        "php" : ">=7.3|8.*"
```

Comment:
Drop support for php 7.x, keep 8.x only 

We will release a 3.0.0

Reply:
9eb4921abd0e439f87aeb6a46bfd00458f01b7cd